elibrary-micro/docker-compose.yml

```diff
@@ -0,0 +1,19 @@
+version: '3.1'
+services:
+  app:
+    container_name: app-springboot-postgresql
+    image: app-springboot-postgresql
+    build: ./
+    ports:
+      - "8085:8080"
+    depends_on:
+      - dbpostgresql
+  dbpostgresql:
+    image: postgres
+    ports:
+      - "5432:5432"
+    environment:
+      - POSTGRES_PASSWORD=password
+      - POSTGRES_USER=postgres
+      - POSTGRES_DB=jfs-operations
```

Comment:
Please change `POSTGRES_DB` to `elibrary-micro`.

Reply:
done